test/test_torch.py

```diff
@@ -6153,6 +6153,17 @@ def test_pickle(self):
         b = pickle.loads(serialized)
         self.assertEqual(a, b)
 
+    def test_pickle_parameter(self):
+        if sys.version_info[0] == 2:
+            import cPickle as pickle
+        else:
+            import pickle
+        a = torch.nn.Parameter(torch.randn(5, 5))
```

Comment:
Could you add an additional test on a Parameter with requires_grad=False, and check that it still loads correctly, i.e., gets you requires_grad=False?